src/ontology/cl-edit.owl

```diff
@@ -28818,14 +28821,16 @@ AnnotationAssertion(oboInOwl:id obo:CL_2000081 "CL:2000081")
 AnnotationAssertion(rdfs:label obo:CL_2000081 "melanocyte of skin of face")
 EquivalentClasses(obo:CL_2000081 ObjectIntersectionOf(obo:CL_1000458 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_1000021)))
 
-# Class: obo:CL_2000082 (melanocyte of foreskin)
+# Class: obo:CL_2000082 (obsolete melanocyte of foreskin)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:TermGenie") obo:IAO_0000115 obo:CL_2000082 "Any melanocyte of skin that is part of a skin of prepuce of penis.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:TermGenie") obo:IAO_0000115 obo:CL_2000082 "OBSOLETE. Any melanocyte of skin that is part of a skin of prepuce of penis.")
+AnnotationAssertion(obo:IAO_0100001 obo:CL_2000082 obo:CL_2000045)
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_2000082 <https://www.wikidata.org/entity/Q35563349>)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_2000082 "2014-12-02T19:10:45Z")
 AnnotationAssertion(oboInOwl:id obo:CL_2000082 "CL:2000082")
-AnnotationAssertion(rdfs:label obo:CL_2000082 "melanocyte of foreskin")
-EquivalentClasses(obo:CL_2000082 ObjectIntersectionOf(obo:CL_1000458 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001471)))
+AnnotationAssertion(rdfs:comment obo:CL_2000082 "Term was duplicated and merged into this term foreskin melanocyte CL_2000045 https://github.com/obophenotype/cell-ontology/issues/1562")
```

Comment:
It would be good to remove the github issue url from the comment and add it in the term_tracker_item annotation.

Reply:
Thank you. I believe I have added the GitHub issue URL in the term_tracker_item annotation as IRI. I am not sure how to remove it from the comment though because I have added this comment in the process of obsoleting the old term using https://oboacademy.github.io/obook/howto/merge-terms/. Could you please guide me through how to remove the github issue url from the comment?

Reply:
I can help with that on a zoom call.